protobuf/lib/src/protobuf/field_type.dart

```diff
@@ -3,31 +3,36 @@
 // BSD-style license that can be found in the LICENSE file.
 
 // ignore_for_file: constant_identifier_names,non_constant_identifier_names
-part of '../../protobuf.dart';
+part of 'internal.dart';
 
-bool _isRepeated(int fieldType) => (fieldType & PbFieldType._REPEATED_BIT) != 0;
+bool _isRepeated(int fieldType) =>
+    (fieldType & PbFieldTypeInternal._REPEATED_BIT) != 0;
 
-bool _isRequired(int fieldType) => (fieldType & PbFieldType._REQUIRED_BIT) != 0;
+bool _isRequired(int fieldType) =>
+    (fieldType & PbFieldTypeInternal._REQUIRED_BIT) != 0;
 
 bool _isEnum(int fieldType) =>
-    PbFieldType._baseType(fieldType) == PbFieldType._ENUM_BIT;
+    PbFieldTypeInternal._baseType(fieldType) == PbFieldTypeInternal._ENUM_BIT;
 
 bool _isBytes(int fieldType) =>
-    PbFieldType._baseType(fieldType) == PbFieldType._BYTES_BIT;
+    PbFieldTypeInternal._baseType(fieldType) == PbFieldTypeInternal._BYTES_BIT;
 
 bool _isGroupOrMessage(int fieldType) =>
-    (fieldType & (PbFieldType._GROUP_BIT | PbFieldType._MESSAGE_BIT)) != 0;
+    (fieldType &
+        (PbFieldTypeInternal._GROUP_BIT | PbFieldTypeInternal._MESSAGE_BIT)) !=
+    0;
 
-bool _isMapField(int fieldType) => (fieldType & PbFieldType._MAP_BIT) != 0;
+bool _isMapField(int fieldType) =>
+    (fieldType & PbFieldTypeInternal._MAP_BIT) != 0;
 
 /// Defines constants and functions for dealing with fieldType bits.
-class PbFieldType {
+class PbFieldTypeInternal {
```

Comment:
This _may_ be the only breaking change in the PR? I.e., we'd need to rev. major versions for it. Not sure if that's what you're going for, or, how important this particular rename is.